snuba/cli/jobs.py

```diff
@@ -11,18 +15,41 @@ def jobs() -> None:
 
 
 @jobs.command()
-@click.argument("job_name")
+@click.option("--json_manifest", required=True)
+@click.option("--job_id")
 @click.option(
     "--dry_run",
     default=True,
 )
-@click.argument("pairs", nargs=-1)
-def run(*, job_name: str, dry_run: bool, pairs: Tuple[str, ...]) -> None:
+def run_from_manifest(*, json_manifest: str, job_id: str, dry_run: bool) -> None:
+    job_specs = _ManifestReader.read(json_manifest)
+    if job_id not in job_specs.keys():
+        raise click.ClickException("Provide a valid job id")
+
+    job_to_run = JobLoader.get_job_instance(job_specs[job_id], dry_run)
+    job_to_run.execute()
 
-    kwargs = {}
+
+def _parse_params(pairs: Tuple[str, ...]) -> MutableMapping[Any, Any]:
+    params = {}
     for pair in pairs:
         k, v = pair.split("=")
-        kwargs[k] = v
+        params[k] = v
+    return params
```

Comment:
nit: We can use dict comprehension here. Something like this:
`return {k: v for k, v in (pair.split("=") for pair in pairs)}`